internal/configuration/schema/access_control.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // ACLRule represent one ACL rule
 type ACLRule struct {
-	Domain    string   `mapstructure:"domain"`
+	Domains   []string `mapstructure:"domain,weak"`
```

Comment:
Please add a comment to justify why we use `weak` here. From my understanding it's for being backward-compatible, isn't it?

Reply:
You're correct, having `weak` here allows someone to still define a single value and coerces it into a string slice. I thought this would be best to make the change as minimally invasive to what the docs currently show and how people have been using Authelia to date.

From what I can tell from [mapstructure's documentation](https://godoc.org/github.com/mitchellh/mapstructure#DecoderConfig.WeaklyTypedInput) decoding should otherwise function similarly since the destination type is a string.

Reply:
Excellent, so just add a comment so that people will keep that in sight while reading the code.